src/FloatingWindows/FLWIntegration.cpp

```diff
@@ -724,6 +729,26 @@ bool FindAndQuarantine (lua_State *L)
   flywithlua::DebugLua();
 }
 
+void LoadImguiFonts() {
+  flywithlua::logMsg(logToDevCon, "FlyWithLua Info: LoadImguiFonts()");
+  // I am trying to just get the fonts to load but big roadblock so far.
```

Comment:
This has to be called after imgui context is created and set, which is currently done here: https://github.com/X-Friese/FlyWithLua/blob/master/src/FloatingWindows/ImGUIIntegration.cpp#L24

Since it is done per window we wither have to load fonts for every window created or make all windows share the same imgui context.